spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/DSLMessage.java

```diff
@@ -101,7 +101,10 @@
 	COMPOSED_TASK_UNQUOTED_TRANSITION_CHECK_MUST_BE_NUMBER(ERROR, 154, "transition conditions must be quoted literals, numerics or '*' but ''{0}'' is not"), //
 	COMPOSED_TASK_NO_LABELS_ON_PARENS(ERROR, 155, "labels cannot be specified on parenthesized groups"), //
 	COMPOSED_TASK_NO_DOUBLE_LABELS(ERROR, 156, "Double labels are not supported"), //
-	COMPOSED_TASK_ARROW_SHOULD_BE_PRECEDED_BY_CODE(ERROR, 157, "Transition arrow must be preceeded by the exit code that should drive the transition")
+	COMPOSED_TASK_ARROW_SHOULD_BE_PRECEDED_BY_CODE(ERROR, 157, "Transition arrow must be preceeded by the exit code that should drive the transition"), //
+	CT_VALIDATION_SECONDARY_SEQUENCES_MUST_BE_NAMED(ERROR, 158, "Secondary sequences must have labels or are unreachable"), //
```

Comment:
Pretty sure this is to force the formatter to have each value on a new line, should leave as is...